container-images/tcib/base/openstackclient/openstackclient.yaml

```diff
@@ -16,11 +16,16 @@ tcib_packages:
   - python3-heatclient
   - python3-ironicclient
   - python3-manilaclient
-  - python3-observabilityclient
   - python3-octaviaclient
   - python3-aodhclient
   - bash-completion
   - iputils
+  #rhel: We can uncomment this when we have rhel specific pkgs
+  x86_64:
```

Comment:
(this one can be removed as discussed yesterday, we don't want/need to distinguish on cpu arch here)